doc/rst/users/build.rst

```diff
@@ -34,9 +45,34 @@ package manager (https://github.com/spack/spack).
 SCR and all of its dependencies exist in a Spack package. After downloading
 Spack, simply type::
 
-  spack install scr
+  spack compiler find --scope site
+  spack install scr@develop resource_manager=SLURM
+
+This will install the component libraries, DTCMP, LWGRP, libyogrt, and pdsh packages (and possibly an MPI and a
+C compiler if needed).
+
+By default, Spack will attempt to build all dependencies for SCR, including many packages such as MPI and OpenSSL that are already installed on many systems. Often it is preferrable to use the pre-installed versions. Spack uses file called 'packages.yaml' to tell it where to find external packages. Information on setting up 'packages.yaml' is available in the Spack documentation (see https://spack.readthedocs.io/en/latest/configuration.html). Spack searches for the packages.yaml file in several locations. When installing Spack in user space, customizations can be placed in $SPACK/etc/packages.yaml or $HOME/.spack/packages.yaml.
+
+Here is an example of a simple `packages.yaml` file::
+
+    packages:
```

Comment:
Let's include representative entries for both SLURM and LSF.

Reply:
put slurm and lsf info for libyogrt in.